src/test/java/org/jitsi/meet/test/base/Participant.java

```diff
@@ -490,17 +490,7 @@ public void saveHtmlSource(File outputDir, String fileName)
      */
     public abstract String getRTPStats();
 
-    public LogEntries getBrowserLogs()
-    {
-        if (type.isFirefox())
-        {
-            // not currently supported in FF
-            // https://github.com/SeleniumHQ/selenium/issues/2910
-            return null;
-        }
-
-        return driver.manage().logs().get(LogType.BROWSER);
-    }
+    public abstract List getBrowserLogs();
```

Comment:
can we specify what kind of list that is ? List<Object> or whatever

Reply:
sorry, I meant `List<Object>`

Reply:
What is the point of having that?

Reply:
Because in one doc you write it's a string in other there can be just anything and it's not obvious what's there. It's just kind of self documenting, but you can leave it as is if you don't like that

Reply:
yeah, maybe I just got used to templates somehow